processing/threedidepth_algorithm.py

```diff
@@ -14,14 +14,20 @@
 from processing.gui.NumberInputPanel import NumberInputPanel
 from processing.gui.wrappers import DIALOG_STANDARD
 from processing.gui.wrappers import WidgetWrapper
-from qgis.core import QgsFeedback
-from qgis.core import QgsProcessingAlgorithm
-from qgis.core import QgsProcessingParameterBoolean
-from qgis.core import QgsProcessingParameterEnum
-from qgis.core import QgsProcessingParameterFile
-from qgis.core import QgsProcessingParameterNumber
-from qgis.core import QgsProcessingParameterRasterDestination
-from qgis.core import QgsProcessingParameterRasterLayer
+from qgis.core import (
+    QgsFeedback,
+    QgsProcessingAlgorithm,
+    QgsProcessingContext,
+    QgsProcessingException,
+    QgsProcessingParameterBoolean,
+    QgsProcessingParameterEnum,
+    QgsProcessingParameterFile,
+    QgsProcessingParameterNumber,
+    QgsProcessingParameterRasterLayer,
+    QgsProcessingParameterString,
+    QgsRasterLayer,
+    QgsMeshLayer,
+)
```

Comment:
This is not the import style that we want.

- We want to be able to use grep, single imports on a single line help with that.
- "make beautiful" also runs "isort", which will change your style back :-)

Reply:
Done.